agenda/domain/src/main/java/com/nibalk/tasky/agenda/domain/usecase/DeleteEventUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package com.nibalk.tasky.agenda.domain.usecase
+
+import com.nibalk.tasky.agenda.domain.EventRepository
+
+class DeleteEventUseCase(
+    private val eventRepository: EventRepository
+) {
+    suspend operator fun invoke(eventId: String) {
+        return eventRepository.deleteEvent(eventId)
+    }
+}
```

Comment:
Do you really gain anything by having these empty use cases that just delegate a call to the repo?